catalog/task/add-fbc-fragment-to-index-image/0.1/add-fbc-fragment-to-index-image.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: add-fbc-fragment-to-index-image
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Submit a build request to add operator bundles to an index image
+  params:
+    - name: BINARY_IMAGE
+      type: string
+      description: FBC Image to be added to index image
+    - name: IIB_SERVICE_URL
+      type: string
+      default: "https://iib.engineering.redhat.com/api/v1"
+      description: IIB service URL
+    - name: IIB_SERVICE_SECRET
+      type: string
+      default: "iib-service-secret"
+      description: Secret with IIB credentials to be used
+    - name: FBC_FRAGMENT
+      type: string
+      description: FBC fragment built by HACBS
+    - name: FROM_INDEX
+      type: string
+      description: Index image (catalog of catalogs) the FBC fragment will be added to    
+    - name: OVERWRITE_FROM_INDEX
+      type: string
+      description: Boolean indicating if the from_index should be overwritten      
+    - name: BUILD_TAGS      
+      type: string
+      description: List of additional tags the internal index image copy should be tagged with
+    - name: BUILD_TIMEOUT_SECONDS
+      type: string
+      default: "300"
+      description:  Timeout seconds to receive the build state
+  results:
+    - name: json_build_info
+      description: JSON build information for the requested build
+    - name: build_state
+      description: IIB Service build state
+  steps:
+    - name: add-fbc-fragment-to-index-image
+      image: quay.io/hacbs-release/release-utils:latest
```

Comment:
I still think the change to the Dockerfile should be its own PR so we can tag that image and not have to merge this with latest